projects/js-packages/publicize-components/src/components/services/custom-inputs.tsx

```diff
@@ -19,9 +20,34 @@ type CustomInputsProps = {
  */
 export function CustomInputs( { service }: CustomInputsProps ) {
 	const id = useId();
+	const [ handleError, setHandleError ] = useState< string | null >( null );
 
 	const reconnectingAccount = useSelect( select => select( store ).getReconnectingAccount(), [] );
 
+	const validateBskyHandle = useCallback( ( value: string ) => {
+		if ( value.endsWith( '.bsky.social' ) ) {
+			const username = value.replace( '.bsky.social', '' );
+			if ( username.includes( '.' ) ) {
+				setHandleError(
+					__(
+						'Bluesky usernames cannot contain dots. If you are using a custom domain, enter it without .bsky.social',
+						'jetpack-publicize-components'
+					)
+				);
```

Comment:
It's better to separate that domain part our of the string and use a placeholder.

```suggestion
				setHandleError(
					sprintf(
						/* translators: %s is the handle suffix like .bsky.social */
						__(
							'Bluesky usernames cannot contain dots. If you are using a custom domain, enter it without "%s"',
							'jetpack-publicize-components'
						),
						'.bsky.social'
					)
				);
```

That quote will make it little more clear what we actually mean.

<img width="488" alt="image" src="https://github.com/user-attachments/assets/a07b4e1d-5b0f-4726-9c69-cd168555d5f4" />